### ISSUE TITLE
How to write player positional data?

### ISSUE THREAD

**Issue by raimanya** [shlifedev/AmongUsMemory#42]
Can someone provide me with the code to change the players position? I've tried rewriting the WriteMemory_IsImposter() multiple times now to make it change the player position, but with no success. Please someone send me a a WriteMemory_Position() method because this is driving me insane lmao

**Comment by randomcmd** [comment]
I was thinking the same thing!

**Comment by Devilx86** [comment]
If you want to change the coordinates of a player including yourself, invoke rpcSnapTo.
Here is how I did it on my menu: https://github.com/Devilx86/ImpostorModmenu/blob/bb6416d4f0925cbab56da80964dba563ba07f709/app/src/main/cpp/mods.h#L383

**Comment by shlifedev** [comment]
```cs

    public void WriteMemory_PositionX(float value)
        {
           
                int _offset_vec2_position = 80;
                var netTransform = ((int)Instance.NetTransform + _offset_vec2_position);
                var targetPointer = netTransform + 4;  
                Cheese.mem.WriteMemory(targetPointer.GetAddress(), "float", value.ToString());
         
        }
```

But you can't change real player position. it's network received value.

**Comment by vladimirdabic** [comment]
But are you able to edit the local player's position? I'm guessing the client (you) is sending your position so changing it shouldn't be a problem.

**Comment by DiamondMiner88** [comment]
Yes you can, but if the server has checks for player positions, teleporting will not work.
But if not, the player position will not change server-side until you send a position update packet.

**Comment by pythonmcpi** [comment]
Note that the server side anticheat now bans you for hacking if you teleport across the map and kill (at least in my case, although I was using a different mod which may be using a different method)
